pom.xml

```diff
@@ -141,6 +140,10 @@
                             <goal>xsdtojava</goal>
                         </goals>
                         <configuration>
+                            <extensions>
+                                <extension>org.jvnet.jaxb2_commons:jaxb2-value-constructor:3.0</extension>
+                                <extension>org.jvnet.jaxb2_commons:jaxb2-basics-runtime:1.11.1</extension>
+                            </extensions>
```

Comment:
can you explain what these extensions do?

I've tried removing them and rebuilding and everything seems to work fine without them